### PR TITLE
Update CIs

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Needs this [cairo PR](https://github.com/gtk-rs/cairo/pull/158) to get merged to work.

cc @EPashkin

### CODE REVIEW

File: src/application.rs [hunk 1]
```diff
@@ -6,7 +6,7 @@ use rt;
 use glib;
 
 impl Application {
-    #[cfg(feature = "v3_6")]
+    #[cfg(any(feature = "v3_6", feature = "dox"))]
```
Comment (EPashkin) [inline src/application.rs]: There dox unneeded

**Comment by EPashkin** [comment]
Also need `dox` for `socket`s in lib.rs

File: .travis.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@ script:
   - cargo test --features embed-lgpl-docs --jobs 1
   # catch any sneaked in lgpl docs
   - cargo build --features purge-lgpl-docs --jobs 1
+  - cargo doc --features "dox"
```
Comment (EPashkin) [inline .travis.yml]: Better move this one line down, as it middle of `# catch any sneaked in lgpl docs` part

File: appveyor.yml [hunk 3]
```diff
@@ -22,5 +22,6 @@ build_script:
   - cd _examples
   - cargo build
   - cargo build --features gtk_3_18
+  - cargo doc --features "dox"
```
Comment (EPashkin) [inline appveyor.yml]: This attempt build examples, not gtk

**Comment by EPashkin** [comment]
https://github.com/gtk-rs/gdk/pull/193 also needed before this.

**Comment by EPashkin** [comment]
*sorry, don't see that gdk's merged

File: src/application.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply src/application.rs]: Oh unneeded. Good catch!

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
IMHO Travis error legit :(

**Comment by EPashkin** [comment]
This part of https://github.com/gtk-rs/gir/pull/467 too and rival of https://github.com/gtk-rs/gtk/pull/583

@GuillaumeGomez ping about travis error.

**Comment by GuillaumeGomez** [comment]
It's legit (at least it was). Let's restart it.

**Comment by GuillaumeGomez** [comment]
Ah damn, `pango` needs to be fixed as well.

**Comment by GuillaumeGomez** [comment]
Ah apparently it was just no up to date...

**Pull request merged**